Adding a related resource for queue-based integration (#4219)
MicrosoftDocs/architecture-center: docs/example-scenario/integration/queues-events-content.md

- [Basic enterprise integration on Azure](../../reference-architectures/enterprise-integration/basic-enterprise-integration.yml)
- [Enterprise business intelligence](/azure/architecture/example-scenario/analytics/enterprise-bi-synapse)
- [Custom Business Processes](../../solution-ideas/articles/custom-business-processes.yml)

> - [A code-centric tutorial showing the use of messaging for integrating with 3rd party systems - using NServiceBus](https://docs.particular.net/tutorials/nservicebus-sagas/3-integration/)

Reviewer: ```suggestion
- [A code-centric tutorial that uses messaging for integrating 3rd party systems using NServiceBus](https://docs.particular.net/tutorials/nservicebus-sagas/3-integration/)
```
